sw/device/lib/dif/dif_base.c

```diff
@@ -0,0 +1,21 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/dif/dif_base.h"
+
+#include <stdbool.h>
+
+bool toggle_to_bool(dif_toggle_t val) {
+  if (val == kDifToggleDisabled) {
```

Comment:
This implements a quiet default case if val != kDifToggleDisabled and val != kDifToggleEnabled. I'd prefer to see it explicitly handled or at least recognized in the source with something like a switch statement or an else statement, to guard against bad behavior if the implementation of a dif_toggle_t were to change in the future.

Reply:
Fixed.